main.js

```diff
@@ -78,7 +78,7 @@ function createWindow(commandLineArguments, workingDirectory) {
        {
            label: 'Redo',
            click: (item, focusedWindow) => {
-               mainWindow.webContents.send("menu-item-click", "\<C-r>")
+               mainWindow.webContents.send("menu-item-click", "\\<C-r>")
```

Comment:
Looks like the changes from the menu change slipped in here too :) You might want to pull the latest master and merge or rebase those changes in.